test/parallel/test-stringbytes-external.js

```diff
@@ -126,12 +122,19 @@ var PRE_3OF4_APEX = Math.ceil((EXTERN_APEX / 4) * 3) - RADIOS;
     return;
   }
 
+  const buf1 = new Buffer(kStringMaxLength + 1);
+  const buf2 = new Buffer(kStringMaxLength);
```

Comment:
This is where the try/catch needs to be. The allocation itself can cause the test to fail. If it fails then return early.